nntrainer/models/neuralnet.cpp

```diff
@@ -241,17 +240,29 @@ int NeuralNetwork::initialize() {
         }
 
         l.setInputDimension(in_layer.getOutputDimension()[location], i);
+      }
 
-        l.net_input[i] = n_buffer;
+      manager.TrackLayerInOuts(l.getName(), l.getInputDimension());
```

Comment:
How about `trackLayerInOuts` to make method name consistent?

Reply:
done.